rustworkx/visualization/matplotlib.py

```diff
@@ -754,6 +754,15 @@ def _connectionstyle(posA, posB, *args, **kwargs):
         else:
             line_width = width
 
+        # radius of edges
+        reverse_edge = [dst, src]
+        if (
+            len(np.where(np.all(edge_pos == reverse_edge, axis=(1, 2)))[0]) != 0
```

Comment:
So this scales quadratically on the number of edges. `O(E^2)`  is not ideal, if we plot a dense graph this is going to be a problem. You need to rewrite this condition using a `set` or something similar instead of naively looking through all the edges